src/main/resources/resources/logisim/strings/std/std.properties

```diff
@@ -799,6 +799,7 @@ TTL7432 = 7432: quad 2-input OR gate
 TTL7434 = 7434: hex buffer gate
 TTL7436 = 7436: quad 2-input NOR gate
 TTL74377 = 74377: octal D-Flipflop with enable
+TTL74381 = 74381: arithmetic logic unit
```

Comment:
I wonder if adding `ALU` to the description (like `74381: arithmetic logic unit (ALU)`) would help (esp. in translated variants). Thoughts?

Reply:
Done